src/util/timemanager.cpp

```diff
@@ -868,4 +868,15 @@ double TimeManager::previousApplicationTimeForInterpolation() const {
     return _previousApplicationTime;
 }
 
+void TimeManager::setFromProfile_time(const Profile& p) {
+    Time t;
+
+    if (p.time->type == Profile::Time::Type::Relative) {
```

Comment:
->
```
switch (p.time->type) {
    case Profile::Time::Type::Relative:
      ....;
    case Profile::Time::Type::Absolute:
     ....;
     default:  throw ghoul::MissingCaseException();
```

That way we'll get a hard error if we introduce a different time specification at some point down the line

Reply:
Made this change